packages/language/src/v1beta1/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var languageServiceApi = require('./language_service_api');
+
+function v1beta1(options) {
```

Comment:
That should be something like:
```javascript
var publisherApi = require('./publisher_api');
var subscriberApi = require('./subscriber_api');
var extend = require('extend');
var gax = require('google-gax');

function v1(options) {
  options = options || {};
  if (!('scopes' in options)) {
    // might be good to use lodash.union?
    options.scopes = [];
    options.scopes.push.apply(options.scopes, publisherApi.ALL_SCOPES);
    options.scopes.push.apply(options.scopes, subscriberApi.ALL_SCOPES);
  }
  var gaxGrpc = gax.grpc(options);
  var result = {};
  extend(result, publisherApi(gaxGrpc));
  extend(result, subscriberApi(gaxGrpc));
  return result;
}
v1.SERVICE_ADDRESS = publisherApi.SERVICE_ADDRESS;
module.exports = v1;
```

notes:
- server_address is common among files, simply picking up one is fine
- scopes might be different, needs to be merged
- the second `extend` might replace some of `publisherApi()` result, but essentially they point to the same structure, so this would be fine